lab/iam/policies/labs4.yaml

```diff
@@ -0,0 +1,33 @@
+Version: "2012-10-17"
+Statement:
+  - Effect: Allow
+    Action:
+      - fsx:CreateFileSystem
```

Comment:
Is it possible to lock some of these down a little bit, for example using `RequestTag` like here https://github.com/aws-samples/eks-workshop-v2/blob/cf4ccfdcf2ced521bf38b21ca5a6a91344ca04d8/lab/iam/policies/labs1.yaml#L10